config/schedule.rb

```diff
@@ -7,6 +7,11 @@
   rake "multicoops:run TASK=foodsoft:notify_users_of_weekly_task"
 end
 
+# Import and assign bank transactions
+every :weekday, :at => %w(5:56am 6:04pm) do
```

Comment:
Just to check: does this mean between these times or twice a day?

Reply:
twice a day

Reply:
perhaps add `if FoodsoftConfig[:use_bank_accounts]` before scheduling this.